### PR TITLE
Create shlok.json

### PR DESCRIPTION
**Pull request by shlok-madhekar**
### Requirements
- [x] You're not using Vercel or Netlify.
- [x] You have completed your website, there's no type of placeholder at the website. *This requirement is optional if you are registering a domain for emails.*
- [x] The website is reachable.
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.
   - *You should have your email presented. If you don't want to share email, you can leave email an empty string (`""`) and add any other social such as Discord/Twitter/etc.*

### Link to Website

https://shlok.subscribe2.repl.co

### CODE REVIEW

**Review by wdhdev — APPROVED** [review]

**Review by andrewstech — DISMISSED** [review]

File: domains/shlok.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "description": "Portfolio Website",
+  "owner": {
+    "username": "Shlok Madhekar",
```
Comment (WilliamDavidHarrison) [inline domains/shlok.json:4]: This needs to be your GitHub username.

@andrewstech this is why the checks were failing.

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: domains/shlok.json (hunk 1 above)
Comment (andrewstech) [reply domains/shlok.json:4]: > This needs to be your GitHub username.
> 
> @andrewstech this is why the checks were failing.

True

Comment (PlaceInSpace) [reply domains/shlok.json]: I just fixed it. Lmk if there is any other issue

**Comment by PlaceInSpace** [comment]
can someone please add the domain

**Review by wdhdev — DISMISSED** [review]

**Review by MaskDuck — DISMISSED** [review]

**Review by MaskDuck — APPROVED** [review]

**Pull request merged**